sql_generators/glean_usage/templates/events_stream_v1.query.sql

```diff
@@ -100,6 +100,11 @@ WITH base AS (
     client_info.client_id AS client_id,
     ping_info.reason AS reason,
     from_map_experiment(ping_info.experiments) AS experiments,
+    {% if has_profile_group_id %}
+      JSON_VALUE(metrics.uuid.legacy_telemetry_profile_group_id) AS profile_group_id,
```

Comment:
Doesn't `JSON_VALUE` here do nothing since it just returns the string as-is?

Reply:
When I tested it in BQ it seemed to return it as JSON so that's why I added it in

Reply:
Although maybe I'm confused - because I thought it was querying off of metrics which is JSON type, but maybe I have the table it's querying wrong? @BenWu 
<img width="751" alt="image" src="https://github.com/user-attachments/assets/50e0d49c-c80b-4b42-b146-9ad88884c99f">

Reply:
Good catch, thanks Ben!